src/components/Amalgamation/BusinessStatus.vue

```diff
@@ -58,16 +58,15 @@ export default class BusinessStatus extends Vue {
           'Form Horizontal amalgamation. '
 
       case AmlStatuses.ERROR_FOREIGN_UNLIMITED:
-        return 'A foreign corporation must not amalgamate with a limited company and continue as ' +
-          'an Unlimited Liability Company.'
+        return 'A foreign corporation must not amalgamate with a BC Company and continue as an ' +
```

Comment:
This is the requirement, "a BC Company of any type cannot amalgamate with a foreign company to form a BC Unlimited Liability Company," which is a join of two sets of rules (which I went over with per Yui).